### ISSUE TITLE
1.0.11 Won't Play Shows

### ISSUE THREAD

**Issue by fsecuree** [micahg/plugin.video.snnow#12]
The Addon won't play any of the sportsnet channels: Ontario, East, West, and Pacific. 360 will Play. I authorize my login with my Shaw Direct Account. Not sure what's causing these channels not to play. It use to work before.

**Comment by micahg** [comment]
Can you confirm that their android app plays all the channels for you.

**Comment by fsecuree** [comment]
Yes my Android phone App does. It's weird. The circle of death just keeps on spinning on my Kodi box.                                                                                                                                                                                                                                                                                                                                                    From: notifications@github.comSent: January 31, 2017 5:21 PMTo: plugin.video.snnow@noreply.github.comReply-to: reply@reply.github.comCc: brianint@gmail.com; author@noreply.github.comSubject: Re: [micahg/plugin.video.snnow] 1.0.11 Won't Play Shows (#12)  Can you confirm that their android app plays all the channels for you.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
